docs/DeveloperGuide.adoc

```diff
@@ -581,29 +581,49 @@ Step 1. The user starts up the application with an initial list loaded from exte
 
 :figure-caption!:
 .Example list on startup
-image::update/updateDG/UpdateStep1.png[]
+image::update/updateDG/UpdateStep1.png[,650]
 
 Step 2. The user inputs **update i/1 n/Government Bursary v/600** to update the 1st transaction's name to "Government Bursary" and value to "600". Input is parsed by `UpdateCommandParser#parse` which creates an `**UpdateCommand**`.
```

Comment:
I think should add `()` at the end of each method

Reply:
Good catch, thanks!